src/page/template/panel/group-participant-user.htm

```diff
@@ -2,7 +2,7 @@
   <!-- ko if: isVisible() -->
     <div class="panel__header">
       <button type="button" class="icon-button" 
-        data-bind="clickOrDrag: onGoBack, attr: { title: t('accessibility.rightPanel.GoBack'), 'aria-label': t('accessibility.rightPanel.GoBack') }"
```

Comment:
do forget to port those changes to the corresponding React component when it is created (ping @przemvs in case you are working on this file)

Reply:
@arjita-mitra  actually already commentd that on the last migration PR, she's on top of this :)

Reply:
🚀